TDAT: public void test bug14609 ( ) throws exception {  <NL> if ( version meets minimum ( 5 , 0 ) ) {  <NL> create table ( " test bug14609 " ,  <NL>  " ( field1 int primary key , field2 decimal ) " ) ;  <NL> this . stmt . execute update ( " insert into test bug14609 values ( 1 , 1 ) " ) ;  <NL>  <NL> prepared statement updatable stmt = this . conn . prepare statement (  <NL>  " select field1 , field2 from test bug14609 " ,  <NL> result set . type _ scroll _ insensitive ,  <NL> result set . concur _ updatable ) ;  <NL>  <NL> try {  <NL> this . rs = updatable stmt . execute query ( ) ;  <NL>  } finally {  <NL> if ( updatable stmt ! = null ) {  <NL> updatable stmt . close ( ) ;  <NL>  }  <NL>  }  <NL>  }  <NL>  }  COM: <s> tests fix for bug 14609 exception thrown for new decimal type when </s>